.gitignore

```diff
@@ -1,32 +1,13 @@
-# See https://help.github.com/articles/ignoring-files for more about ignoring files.
-#
-# If you find yourself ignoring temporary files generated by your text editor
-# or operating system, you probably want to add a global ignore instead:
-#   git config --global core.excludesfile '~/.gitignore_global'
-
-# Ignore bundler config.
-/.bundle
-
-# Ignore all logfiles and tempfiles.
-/log/*
-/tmp/*
-!/log/.keep
-!/tmp/.keep
-
-
-/public/assets
-.byebug_history
+.idea/
+*.iml
+*~
+.bundle/
 .env
-
-# Ignore master key for decrypting credentials and more.
-/config/master.key
-
-/public/packs
-/public/packs-test
-/node_modules
-/yarn-error.log
-yarn-debug.log*
-.yarn-integrity
```

Comment:
Is there a reason on why these were deleted?

Reply:
Um, mostly because apparently my copy was out of date when I committed it. They should DEFINITELY be added back in.

Reply:
Okay, I'm going to revise the branch entirely - give me a few minutes, I'm going to work with a better delta. I'm not sure where the original went on my filesystem.